### PR TITLE
chore(sample): additionnal sample cleanup

### PR DESCRIPTION
**Pull request by freeboub**
## Summary
The goal of this PR is to remove circular dependency in the sample app which is causing a warning at every startup

### Motivation
The goal of this PR is to remove circular dependency in the sample app which is causing a warning at every startup

### Changes
move 2 files in components
rework imports / exports
fix linter warning

## Test plan
test on sample

### CODE REVIEW

File: examples/basic/src/components/Overlay.tsx [hunk 1]
```diff
@@ -23,14 +21,15 @@ import {
...
-} from '../components';
+
+import {toast} from './Toast';
+import {Seeker} from './Seeker';
+import {AudioTrackSelector} from './AudioTracksSelector';
+import {VideoTrackSelector} from './VideoTracksSelector';
+import {TextTrackSelector} from './TextTracksSelector';
+import {TopControl} from './TopControl';
+import {ToggleControl} from './ToggleControl';
+import {MultiValueControl} from './MultiValueControl';
```
Comment (seyedmostafahasani) [inline examples/basic/src/components/Overlay.tsx:32]: I think it's better to import as shown below:
```
import {
  toast,
  Seeker,
  AudioTrackSelector,
  TextTrackSelector,
  VideoTrackSelector,
  TopControl,
  ToggleControl,
   MultiValueControl
} from '../components';
```

**Comment by seyedmostafahasani** [comment]
I just left a small comment.

File: examples/basic/src/components/Overlay.tsx (hunk 1 above)
Comment (freeboub) [reply examples/basic/src/components/Overlay.tsx:32]: But will produce the circular dependency issue.. as ../component/index importss Overlay, Overlay will try to import itself...

Comment (freeboub) [reply examples/basic/src/components/Overlay.tsx:32]: But, maybe overlay shouldn't be in component?!

Comment (seyedmostafahasani) [reply examples/basic/src/components/Overlay.tsx:32]: You're right. I approved it.

**Review by seyedmostafahasani — APPROVED** [review]

**Pull request merged**